src/store.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import { getPersistentStore } from 'pc-nrfconnect-shared';
+
+interface StoreSchema {
+    settings: { autoRead: boolean };
+    mruFiles: string[];
+}
+
+interface Settings {
+    autoRead: boolean;
+}
+
+const { set, get } = getPersistentStore<StoreSchema>();
```

Comment:
Did you verify that this works? I tried doing the same thing in Cellular Monitor, but then things started failing. I didn't want to spend much time on it right now though, so I didn't look closer into it yet.

Reply:
Yes, verified it and it worked nicely. Still works today :)